test/controllers/XiEoriControllerSpec.scala

```diff
@@ -80,6 +82,57 @@ class XiEoriControllerSpec extends SpecBase {
     }
   }
 
+  "getXiEoriInformationV2" should {
+
+    "return xi eori information for the eori, if available in the database" in new Setup {
+      when(mockXiEoriInformationRepository.get(eqTo(TEST_EORI_VALUE)))
+        .thenReturn(Future.successful(Some(xiEoriInformation)))
+
+      running(app) {
+        val request = FakeRequest(GET, getXiEoriInformationV2Route)
+
+        val result = route(app, request).value
+
+        contentAsJson(result).as[XiEoriInformation] mustBe xiEoriInformation
+      }
+    }
+
+    "return xi eori information for the eori, if info is not found in the database but retrieved from Sub09" in new Setup {
+      when(mockXiEoriInformationRepository.get(eqTo(TEST_EORI_VALUE)))
+        .thenReturn(Future.successful(Some(xiEoriInformation)))
```

Comment:
Should this not be returning `None` given the test name?

Reply:
Done, seems was mistakenly missed